website/static/js/registrationUtils.js

```diff
@@ -198,6 +204,18 @@ var Question = function(data, id) {
     });
 
     /**
+     * @returns {Boolean} true if the value <input> does validate
+     **/
+    self.hasErrors = ko.computed(function() {
+        if( self.required ){
```

Comment:
I don't really like the check for self.required. We may need to implement other kinds of validation later and would rather not special case required as a form of validation. 

What about:
```javascript
if (self.value.isValid) {  // if validators attached to this question) 
  return !self.value.isValid();
}
else {
  return false;
}